packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/TaskNavButtons.js

```diff
@@ -8,36 +8,20 @@ import DoneAndTalkButton from './components/DoneAndTalkButton'
 import BackButton from './components/BackButton'
 import ExpertOptions from './components/ExpertOptions'
 
-export default function TaskNavButtons({
-  disabled = false
-}) {
+export default function TaskNavButtons({ disabled = false }) {
   const [saving, setSaving] = useState(false)
 
   return (
-    <Box
-      direction='row'
-      justify='center'
-    >
+    <Box direction='row' justify='center'>
       <BackButton />
-      <NextButton
-        autoFocus={false}
```

Comment:
There's no documentation or explanatory unit tests about why `autoFocus` was applicable to NextButton. I've removed it from this file and NextButton because it seems to never be set to "true", so just cruft.